app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb

```diff
@@ -32,10 +32,10 @@ snippet: true
   options.push("kssendmac", "ks.sendmac", "inst.ks.sendmac")
 
   # networking credentials
-  raise("Dual-stack provisioning not supported") if subnet4 && subnet6
-  if subnet4 && subnet4.dhcp_boot_mode?
+  raise("Both IPv4 and IPv6 subnets are set on a provisioning interface, create 'dualstack' parameter with either 'IPv4' or 'IPv6' to identify which to use.") if subnet4 && subnet6 && !host_param("dualstack").present?
```

Comment:
Minor language suggestions:
```suggestion
  raise("Both IPv4 and IPv6 subnets are set on a provisioning interface, create a 'dualstack' parameter with either 'IPv4' or 'IPv6' to identify which protocol to use.") if subnet4 && subnet6 && !host_param("dualstack").present?
```

However, I still think this is a confusing name. When I read dual stack, I think both IPv4 and IPv6. Thoughts on using `kickstart-protocol=force-ipv6` instead of `dualstack=IPv6`?

Reply:
Amended the change, thanks.